examples/QIR/Development/Development.csproj

```diff
@@ -1,21 +1,21 @@
-﻿<Project Sdk="Microsoft.Quantum.Sdk/0.20.2110171573">
+<Project Sdk="Microsoft.Quantum.Sdk/0.23.198514-beta">
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
     <BuildOutputPath>$(MSBuildThisFileDirectory)build</BuildOutputPath>
     <ExecutablePath>$(BuildOutputPath)\$(MSBuildProjectName)</ExecutablePath>
   </PropertyGroup>
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Quantum.LlvmBindings.Native" Version="13.0.0-CI-20220129-005156" PrivateAssets="All" GeneratePathProperty="true"/>
-    <PackageReference Include="Microsoft.Quantum.Qir.Runtime" Version="0.20.2110171573-alpha" GeneratePathProperty="true" />
-    <PackageReference Include="Microsoft.Quantum.Simulators" Version="0.20.2110171573" GeneratePathProperty="true" />
+    <PackageReference Include="Microsoft.Quantum.Qir.Runtime" Version="0.22.189218-alpha" GeneratePathProperty="true" />
```

Comment:
The other references use `0.23.198514-beta` so I would expect this to use the alpha of the same version if available. Is that not possible?

Reply:
That was a mistake on my part, just fixed it. :)